build.gradle

```diff
@@ -138,6 +138,10 @@ tasks.withType(Checkstyle) {
     exclude 'uk/gov/hmcts/ccd/domain/types/sanitiser/document/**'
 }
 
+tasks.withType(Test) {
+    maxParallelForks = System.getenv('MAX_NUM_PARALLEL_THREADS') ?: 6
```

Comment:
Because of total setup cost of each thread (including repeated new context creations for each thread, if any) 6 threads by default may or may not be optimal. Could you analyse the difference for 2 to 6 threads and report the observations under the Jira ticket?